frontend/src/components/personal/customization/CustomizationForm.tsx

```diff
@@ -0,0 +1,156 @@
+
+  const [checkboxState, setCheckboxState] = useState({
+      sendEmailsWithNews: custom? custom.sendEmailsWithNews : true,
+      sendUpdatesThreads: custom? custom.sendUpdatesThreads: false,
+      sendUserReviews: custom? custom.sendUserReviews : false,
+      sendUserQuestsReviews: custom? custom.sendUserQuestsReviews : false,
+      sendUpdatesMessages: custom? custom.sendUpdatesMessages : false,
+    })
+
+    const [textAndTimeState, setTextAndTimeState] = useState({
+      timezone: custom? custom.timezone :'UTC',
+      aboutUser: custom? custom.aboutUser : ''
+    })
+
+    useEffect(() => {
+      setCheckboxState({
+        sendEmailsWithNews: custom? custom.sendEmailsWithNews : true,
+        sendUpdatesThreads: custom? custom.sendUpdatesThreads: false,
+        sendUserReviews: custom? custom.sendUserReviews : false,
+        sendUserQuestsReviews: custom? custom.sendUserQuestsReviews : false,
+        sendUpdatesMessages: custom? custom.sendUpdatesMessages : false,
+      })
+      setTextAndTimeState({
+        timezone: custom? custom.timezone :'UTC',
+        aboutUser: custom? custom.aboutUser : ''
+      })
+   },[custom])
+    
+    const { sendEmailsWithNews, sendUpdatesThreads, sendUserReviews, sendUserQuestsReviews,  
+      sendUpdatesMessages} = checkboxState
+    const { timezone, aboutUser } = textAndTimeState
+    
+    const handleCheckboxChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+      setCheckboxState({ ...checkboxState, [event.target.name]: event.target.checked })
+    }, [checkboxState, setCheckboxState])
+    
+    const handleTextAndTimeChange = useCallback((event: React.ChangeEvent<{ value: unknown }>) => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      //@ts-ignore
+      setTextAndTimeState({...textAndTimeState, [event.target.name]: event.target.value as string })
+    }, [textAndTimeState, setTextAndTimeState])
+
+    const handleSubmit = useCallback((e) => {
+      e.preventDefault()
+    const customizationData = Object.assign(checkboxState, textAndTimeState)
+    console.log(customizationData)
+    dispatch(successChangeUserCustomization(customizationData))
+    }, [dispatch, checkboxState, textAndTimeState])
+
+    const getTime = () => {
+      const zone = textAndTimeState.timezone
+      const time = new Date()
+      if (zone === 'GMT') {
+        //Make minutes quantity two-digits number even if its below 10
+        const minutes = String(time.getMinutes()).padStart(MINUTES_QTY_TWO_CHARS_LONG, '0')
+        const now = `${time.getHours()} : ${minutes}`
+        return now
+      }
+       //Make minutes quantity two-digits number even if its below 10
+      const minutes = String(time.getUTCMinutes()).padStart(MINUTES_QTY_TWO_CHARS_LONG, '0')
+      const now = `${time.getUTCHours()} : ${minutes}`
+      return now
+    }
+
+    const getTimeDiffLocalTimeWithGMT = () => {
```

Comment:
Let's move `getTime` and `getTimeDiffLocalTimeWithGMT` out of component so it won't get created each render.